devicetypes/automateasia/wyfy-touch-zwave-switch-child.src/wyfy-touch-zwave-switch-child.groovy

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Since Your devices are going to be integrated in zwave-multi-metering-switch.groovy, this file has to be removed.